.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ on:
   pull_request: ~
 
 jobs:
-  build:
+  build and test:
```

Comment:
Please keep this name as `build`, changing the name will not allow this to get merged because the settings are looking for a `build` step to pass.

You also can't give names spaces.